pages/dashboard/project/[id].tsx

```diff
@@ -0,0 +1,184 @@
+
+@observer
+export default class ProjectEvaluationPage extends ObservedComponent<
+  ProjectEvaluationPageProps,
+  typeof i18n
+> {
+  static contextType = I18nContext;
+
+  projectId = +this.props.route!.params!.id;
+
+  evaluationStore = new ConsultMessageModel(this.projectId);
+
+  get menu() {
+    const { t } = this.observedContext;
+
+    return [
+      { href: '/dashboard', title: t('overview') },
+      { href: `/dashboard/project/${this.projectId}`, title: t('project_evaluation') },
+    ];
+  }
+
+  renderChatMessage = ({ id, content, evaluation, createdAt, createdBy }: ConsultMessage) => {
+    const { t } = this.observedContext;
+    const isBot = createdBy.roles.includes(3 as UserRole.Robot);
+    const avatarSrc = isBot ? '/robot-avatar.png' : createdBy?.avatar || '/default-avatar.png';
+    const name = isBot ? `🤖 ${t('ai_assistant')}` : createdBy?.name || 'User';
+
+    return (
+      <Box
+        key={id}
+        sx={{
+          display: 'flex',
+          justifyContent: isBot ? 'flex-start' : 'flex-end',
+          mb: 2,
+          width: '100%',
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: isBot ? 'row' : 'row-reverse',
+            alignItems: 'flex-start',
+            maxWidth: '80%',
+            gap: 1,
+          }}
+        >
+          <Avatar src={avatarSrc} alt={name} sx={{ width: 32, height: 32 }} />
+          <Paper
+            elevation={1}
+            sx={{
+              p: 2,
+              backgroundColor: isBot ? 'grey.100' : 'primary.light',
+              color: isBot ? 'text.primary' : 'primary.contrastText',
+              borderRadius: isBot ? '16px 16px 16px 4px' : '16px 16px 4px 16px',
+            }}
+          >
+            <Typography variant="caption" display="block" sx={{ mb: 0.5, opacity: 0.8 }}>
+              {name}
+            </Typography>
+
+            {content && (
+              <Typography variant="body2" sx={{ mb: 1 }}>
+                {content}
+              </Typography>
+            )}
+            {evaluation && <EvaluationDisplay {...evaluation} />}
+
+            {createdAt && (
+              <Typography variant="caption" sx={{ opacity: 0.6, fontSize: '0.75rem' }}>
+                {new Date(createdAt).toLocaleTimeString()}
+              </Typography>
+            )}
+          </Paper>
+        </Box>
+      </Box>
+    );
+  };
+
+  handleMessageSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const { content } = formToJSON<{ content: string }>(event.currentTarget);
+    
+    if (!content.trim()) return;
+
+    try {
+      await this.evaluationStore.updateOne({ content: content.trim() });
+      // Clear the form
+      event.currentTarget.reset();
+      // Refresh the messages
+      this.evaluationStore.clear();
+      await this.evaluationStore.getList();
+    } catch (error) {
+      console.error('Failed to send message:', error);
+    }
+  };
```

Comment:
```suggestion
  handleMessageSubmit = async (event: FormEvent<HTMLFormElement>) => {
    event.preventDefault();

    let { content } = formToJSON<{ content: string }>(event.currentTarget);

    content = content.trim();

    if (!content) return;

    await this.evaluationStore.updateOne({ content });

    event.currentTarget.reset();
  };
```
1. 逻辑方法放在渲染方法上方
2. 重写 `updateOne()`，以实现本地新增一项，内部核心逻辑为：
    ```ts
    this.restoreList([...this.allItems, newOne]);
    ```
3. 再用 `web-utility` 中的 `scrollTo()` 滚到最后一条消息
4. 最后声明一个 `debounce()` 包裹的方法，去 `POST project/{pid}/consult-message/evaluation` 来触发评估，返回后再新增一条消息